rados/rados_test.go

```diff
@@ -14,11 +14,11 @@ import (
 	"github.com/gofrs/uuid"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"github.com/stretchr/testify/suite"
+	tsuite "github.com/stretchr/testify/suite"
 )
 
 type RadosTestSuite struct {
-	suite.Suite
+	tsuite.Suite
```

Comment:
Again, curious: why?

Reply:
suite was used elsewhere in the file for the actual test suite (generally, the receiver var) so to disambiguate "testify suite" -> "tsuite"